src/panels/config/integrations/integration-panels/zwave_js/zwave_js-custom-param.ts

```diff
@@ -117,21 +117,32 @@ class ZWaveJSCustomParam extends LitElement {
     `;
   }
 
+  private _tryParseNumber(value: string): number | undefined {
+    if (!value) return;
+    const parsed = Number(value);
+    if (Number.isNaN(parsed)) return;
+    // eslint-disable-next-line consistent-return -- not sure why eslint expects no return here
```

Comment:
```suggestion
```

Reply:
Odd - was this not an error on CI? Only locally on my end?